src/main/java/picard/sam/SamToFastq.java

```diff
@@ -93,6 +91,22 @@
             "     I=input.bam<br />" +
             "     FASTQ=output.fastq" +
             "</pre>" +
+            "<h4>Tag Usage example:</h4>" +
+            "<br />" +
+            "This will create the fastq from read sequences alongside two fastq files.  One will be converted from the " +
+            " \"CR\" tag with quality from the \"CY\" tag.  The other fastq will be converted from the \"CB\" and \"UR\" " +
+            "tags concatenated together with no separator(not specified on command line) with the qualities croming from " +
+            "the \"CY\" and \"UY\" tags concatenated together." +
+            "<br />" +
+            "<pre>" +
+            "java -jar picard.jar SamToFastq <br />" +
+            "     I=input.bam<br />" +
+            "     FASTQ=output.fastq<br />" +
+            "     SEQUENCE_TAG_GROUP=CR<br />" +
+            "     QUALITY_TAG_GROUP=CY<br />" +
+            "     SEQUENCE_TAG_GROUP=\"CB,UR\"<br />" +
+            "     QUALITY_TAG_GROUP=\"CY,UY\"" +

```

Comment:
I have no idea what the file names are for the FASTQs in this example without reading the code.

Reply:
agreed, I would hope when someone saw the output it would make sense but I totally understand.